modules/api/src/main/kotlin/dev/proxyfox/api/Authentication.kt

```diff
@@ -15,7 +15,6 @@ import io.ktor.server.response.*
 val AuthenticationPlugin = createRouteScopedPlugin(name = "AuthenticationPlugin") {
     onCall { call ->
         val token = call.request.headers["Authorization"] ?: return@onCall call.respond("401 Unauthorized")
-        val userToken = database.getOrCreateTokenFromSystem(call.parameters["id"]!!).token
-        if (token != userToken) return@onCall call.respond("403 Forbidden")
```

Comment:
One thing I didn't notice with the last review: Does this actually set the HTTP code?

Reply:
Fair enough. Probably something to be on a todo somewhere so it's not forgotten.